### PR TITLE
Improvements to password reset email

### PR DESCRIPTION
**Pull request by KaitCrawford**

### CODE REVIEW

File: mentorship/mentor/templates/reset_password_html_body.html [hunk 1]
```diff
@@ -1,13 +1,18 @@
 {% load i18n %}{% autoescape off %}
+<p>
 {% blocktrans %}You're receiving this email because you requested a password reset for your Mentor2Go account.{% endblocktrans %}
```
Comment (hodgestar) [inline mentorship/mentor/templates/reset_password_html_body.html]: Should this not be a paragraph on its own? I.e. have a closing `</p>` and and opening one for the next paragraph?

File: mentorship/mentor/templates/reset_password_html_body.html [hunk 2]
```diff
@@ -1,13 +1,18 @@
...
 
 {% trans "Please open the Mentor2Go app and provide the following token when prompted:" %}
+<br>
 {% block token %}
 {{ token }}
 {% endblock %}
+</p>
+<p>
+{% blocktrans %}If you are having any trouble resetting your password please email us at m2g@mentortogther.org.{% endblocktrans %}
```
Comment (hodgestar) [inline mentorship/mentor/templates/reset_password_html_body.html]: Wondering is this should be a separate paragraph too.

File: mentorship/mentor/templates/reset_password_plain_body.html [hunk 3]
```diff
@@ -5,6 +5,7 @@
 {% block token %}
 {{ token }}
 {% endblock %}
+{% blocktrans %}If you are having any trouble resetting your password please email us at m2g@mentortogther.org.{% endblocktrans %}
```
Comment (hodgestar) [inline mentorship/mentor/templates/reset_password_plain_body.html]: If this is a separate paragraph in the HTML version, perhaps there should be a line-break above it here?

**Comment by hodgestar** [comment]
Left a bunch of bike-shedding.

**Comment by KaitCrawford** [comment]
@hodgestar ready for re-review

**Comment by hodgestar** [comment]
:+1:

**Pull request merged**